packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -124,6 +149,8 @@ library TaikoData {
         // this block is not verified as the last block in a batch, verifiedTransitionId
         // will remain zero.
         uint32 verifiedTransitionId;
+        uint64 anchorBlockId; // slot 4
+        uint64 anchorTimestamp;
```

Comment:
Not saying my names are necessarily better, but anchorX names are also a bit confusing haha. Because `anchorTimestamp` is never used in the anchor, it's just the timestamp put in the L2 block header. And `anchorBlockId` is not the anchor call's block id, but I guess this one somehow could make more sense to people than my L1 state block name. But I think `anchorTimestamp` can just remain `timestamp`.

Reply:
`anchorTimestamp` renamed to `timestamp`.

Reply:
Oh yes yes it's the value of `_l1BlockId`! Just that you people may think that `anchorBlockId` is the block in which the anchor is included, but I guess always different interpretations possible. :)